tests/torch/sparsity/movement/examples/Quickstart.md

```diff
@@ -0,0 +1,25 @@
+# Movement example runs
```

Comment:
Because it's one of the example of JPQD usage. Unfortunately, it's not purely based on HF examples and written from scratch based on the callback, which might be not convenient for the users to reuse. 
Currently, it's utilized for tests only. 
For users, we envision usage via Optimum (support is coming separately from this PR) or via existing patch-based NNCF integration to HF.